compiler/src/dmd/frontend.h

```diff
@@ -5427,6 +5426,8 @@ extern Type* merge(Type* type);
 
 extern Dsymbol* toDsymbol(Type* type, Scope* sc);
 
+extern Type* trySemantic(Type* type, const Loc& loc, Scope* sc);
+
```

Comment:
this looks to be a duplicate of the line just below it. Messed up a rebase?

Reply:
Nope, one is TRYSemantic whereas the other is TYPESemantic.

Reply:
They do look very similar, though.

Reply:
Ah, never mind then.